### PR TITLE
set package to v1alpha1 for crd

### PR DESCRIPTION
**Pull request by nb950**
# Description
set package to v1alpha1 for crd

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] run unit tests ok
make image ok
deploy driver ok

### CODE REVIEW

File: deploy/operator.yaml [hunk 1]
```diff
@@ -429,7 +429,7 @@ spec:
         - --leader-elect
         command:
         - /manager
-        image: dellemc/dell-csm-operator:v0.1.0
+        image: amaas-eos-mw1.cec.lab.emc.com:5036/dell-csm-operator:v0.0.2
```
Comment (rensyct) [inline deploy/operator.yaml:432]: please revert this change. we should not have references to internal repo

**Review by jooseppi-luna — COMMENTED** [review]
LGTM other than what @rensyct pointed out, once that is fixed I can approve.

File: deploy/operator.yaml (hunk 1 above)
Comment (nb950) [reply deploy/operator.yaml]: done

**Review by jooseppi-luna — APPROVED** [review]
LGTM, thanks!

**Review by rensyct — APPROVED** [review]
